src/main/java/frc/robot/Robot.java

```diff
@@ -7,6 +7,8 @@
 import com.revrobotics.REVPhysicsSim;
 
 import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
 // WPILib Imports
```

Comment:
Let's put the WPILib imports under the WPILib Imports section. You can fix this in the next patchset.